core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -851,3 +851,42 @@ class EventsByTagStrictBySeqNoEarlyFirstOffsetSpec
   }
 }
 
+class EventsByTagStrictBySeqNoManyInCurrentTimeBucketSpec
```

Comment:
@patriknw Unfortunately after some more testing today, I ran again into the same (hanging loop) situation as earlier. Somehow I cannot recreate the situation in a test. Although I think a potential bug is solved with this PR, there still is an issue. It would be really helpful to have an option to switch off lookingForDelayed during replay.